src/runtime/pythonengine.cs

```diff
@@ -151,7 +151,14 @@ public static void InitializeLibrary()
         {
             if (!libraryLoaded)
             {
-                var _loader = Python.Runtime.Platform.LibraryLoader.Get(Runtime.OperatingSystem);
+#if MONO_OSX
```

Comment:
The hope is to transition away from having ifdefs, so we can eventually just ship one single platform-independent DLL.

Reply:
I think that it's been done recently in the upstream